src/codecs/framing/character_delimited.rs

```diff
@@ -214,13 +213,20 @@ mod tests {
         let buf = &mut BytesMut::new();
 
         buf.reserve(200);
-        // limit is 6 so this should fail
+        // limit is 6 so it will skip longer lines
         buf.put_slice(b"1234567\n123456\n123412314\n123");
 
-        assert!(codec.decode(buf).unwrap().is_none());
-        assert!(codec.decode(buf).unwrap().is_some());
-        assert!(codec.decode_eof(buf).unwrap().is_none());
-        assert!(codec.decode_eof(buf).unwrap().is_some());
+        assert_eq!(codec.decode(buf).unwrap(), Some(Bytes::from("123456")));
+        assert_eq!(codec.decode(buf).unwrap(), None);
+
+        let buf = &mut BytesMut::new();
+
+        buf.reserve(200);
```

Comment:
That seems a bit arbitrary / unnecessary in this test. Or am I missing something?

Reply:
Yeah, I'm not really sure why we are reserving. That already existed in the tests, but I'll drop it in both places.